lib/rspecq/worker.rb

```diff
@@ -42,10 +42,10 @@ class Worker
 
     attr_reader :queue
 
-    def initialize(build_id:, worker_id:, redis_host:)
+    def initialize(build_id:, worker_id:, redis_host:, fail_fast: nil)
```

Comment:
Ditto for making this an `attr_accessor` instead (like `populate_timings`, `files_or_dirs_to_run` etc.)

Reply:
966619b